internal/pkg/config/federation_config.go

```diff
@@ -45,6 +47,18 @@ type Remote struct {
 	Network     string      `json:"network"`
 }
 
+func (r *Remote) ServiceName() string {
+	return fmt.Sprintf("federation-discovery-service-%s", r.Name)
+}
+
+func (r *Remote) ServiceFQDN() string {
+	return fmt.Sprintf("federation-discovery-service-%s.istio-system.svc.cluster.local", r.Name)
```

Comment:
Why not:

```suggestion
	return fmt.Sprintf("%s.istio-system.svc.cluster.local", r.ServiceName())
```

Reply:
I also wonder if the namespace shouldn't be resolved dynamically here.

Reply:
> Why not:

Done

> I also wonder if the namespace shouldn't be resolved dynamically here.

We don't know the remote namespace for now, but we will be able to change it when we will have `remote.identity` implemented.